taichi/program/async_utils.cpp

```diff
@@ -194,7 +194,8 @@ TaskMeta *get_task_meta(IRBank *ir_bank, const TaskLaunchRecord &t) {
               ir_bank->get_async_state(child.get(), AsyncState::Type::value));
         }
       } else if (snode_op->op_type == SNodeOpType::is_active ||
-                 snode_op->op_type == SNodeOpType::length) {
+                 snode_op->op_type == SNodeOpType::length ||
+                 snode_op->op_type == SNodeOpType::get_addr) {
         meta.input_states.insert(
             ir_bank->get_async_state(sn, AsyncState::Type::mask));
       } else {
```

Comment:
Actually `SNodeOpType::get_addr` does not require an `mask` state in the async graph.

It's better to simply ignore the op here. Something like

```C++
...
} else if (snode_op->op_type == SNodeOpType::get_addr) {
    // do nothing
}
...
```

Reply:
I see. thanks.